go/libkb/json.go

```diff
@@ -131,9 +153,35 @@ func newJSONFileTransaction(f *JSONFile) (*jsonFileTransaction, error) {
 }
 
 func (f *JSONFile) Save() error {
+	tx, txCreated, err := f.getOrMakeTx()
```

Comment:
Can this function be simplified at all?  since it's only a 3-liner, maybe don't use a `defer`?